sites/main-footer/inc-footer.html

```diff
@@ -1,25 +1,29 @@
-<div class="pagedetails{% if include.addContainer %} container{% endif %}">
-	<div class="row">
-		<div class="col-sm-6 col-md-5 col-lg-4">
-		{%- unless page.noReportProblem -%}
-				<details class="brdr-0">
-					<summary class="btn btn-default text-center">{{ i18nText-feedback }}</summary>
-					<div class="well row">
-						{%- if i18nText-lang == "fr" -%}
-							{% include main-footer/inc-report-problem-fr.html %}
-						{%- elsif i18nText-lang == "en" -%}
-							{% include main-footer/inc-report-problem-en.html %}
-						{%- endif -%}
-					</div>
-			</details>
-		{%- endunless -%}
-		</div>
+<footer class="pagedetails{% if include.addContainer %} container{% endif %}">
+	<h2 class="wb-inv">Report a problem of this page or share it</h2>
```

Comment:
@duboisp In my opinion, something as simple as `Page details` would make sense. En français, je ne sais pas ce qui a été utilisé en tant normal mais peut-être `Détails de la page`?

Reply:
@GormFrank I am fine with that. We can always adjust later in a few months, it will only generate a major but easy change to GCWeb.

 May be @delisma have a naming suggestion for the heading?

Reply:
Page details / Détails de la page is used